mlir/lib/Conversion/TosaToLinalg/TosaToLinalg.cpp

```diff
@@ -82,13 +82,16 @@ materializeBinaryNanCheckIfRequired(OpTy op, PatternRewriter &rewriter,
                                           rhsOrResult);
 }
 
-template <typename T>
+// Create i32 Value from zp, a zero-point value sign-extended from bitwidth.
```

Comment:
Find the brief and name of the function a bit confusing meaning that it doesn't clearly convey what is going on.